python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/graphql/listDataProductAssets.graphql

```diff
@@ -0,0 +1,148 @@
+query search($urn: String!, $query: String!, $count: Int!, $start: Int!) {
```

Comment:
The `search` here is just a name. I'm not sure where this would actually be surfaced but we should change it to reflect what the query does, just to avoid confusion with the other query

Reply:
yeah good point, will change